examples/awel/simple_nl_schema_sql_chart_example.py

```diff
@@ -0,0 +1,181 @@
+import os
+from typing import Dict
+
+from pydantic import BaseModel, Field
+
+from dbgpt.configs.model_config import MODEL_PATH, PILOT_PATH
+from dbgpt.core.awel import DAG, HttpTrigger, JoinOperator, MapOperator
+from dbgpt.datasource.rdbms.conn_sqlite import SQLiteTempConnect
+from dbgpt.model import OpenAILLMClient
+from dbgpt.rag.embedding.embedding_factory import DefaultEmbeddingFactory
+from dbgpt.rag.operator.chart_draw import ChartDrawOperator
+from dbgpt.rag.operator.schema_linking import SchemaLinkingOperator
+from dbgpt.rag.operator.sql_gen import SqlGenOperator
+from dbgpt.storage.vector_store.chroma_store import ChromaVectorConfig
+from dbgpt.storage.vector_store.connector import VectorStoreConnector
+
+"""AWEL: Simple nl-schemalinking-sql-chart operator example
+
+    pre-requirements:
+        1. install openai python sdk
+        ```
+            pip install openai
```

Comment:
Great! May be `pip install -e "[.openai]"` here~

Reply:
Thanks for your suggestion, I will revise it again